src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: bold;
+}
+
+.position-block-search {
+  position: relative;
+  margin: 20px 0;
+}
+
+.icon-search {
+  position: absolute;
+  z-index: 1;
+}
+
+.search {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  font-weight: 300;
+  outline: none;
+  font-family: Avenir, sans-serif;
```

Comment:
assign font family in body tag

Reply:
if i write font-family in the body then it will not work in the input